6-AdvancedScenarios/4-hybrid-SPA/README.md

```diff
@@ -0,0 +1,409 @@
+
+  1. On Windows, run PowerShell as **Administrator** and navigate to the root of the cloned directory
+
+  1. In PowerShell run:
+
+    ```PowerShell
+    Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+    ```
+
+  1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+  1. For interactive process - in PowerShell run:
+
+    ```PowerShell
+      cd .\AppCreationScripts\
+      .\Configure.ps1 -TenantId "[Optional] - your tenant id" -Environment "[Optional] - Azure environment, defaults to 'Global'"
+     ```
+
+  > Other ways of running the scripts are described in [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md)
+  > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+ </details>
+
+### Choose the Azure AD tenant where you want to create your applications
+
+1. Sign in to the [Azure portal](https://portal.azure.com).
+1. If your account is present in more than one Azure AD tenant, select your profile at the top right corner in the menu on top of the page, and then **switch directory** to change your portal session to the desired Azure AD tenant.
+
+### Register the service app (msal-hybrid-spa)
+
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD** service.
+1. Select the **App Registrations** blade on the left, then select **New registration**.
+1. In the **Register an application page** that appears, enter your application's registration information:
+   * In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-hybrid-spa`.
+   * Under **Supported account types**, select **Accounts in this organizational directory only**.
+   * In the **Redirect URI (optional)** section, select **Web** in the combo-box and enter the following redirect URI: `http://localhost:5000/redirect`.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select **Authentication** in the menu.
+   * If you don't have a platform added, select **Add a platform** and select the **Web** option.
+   * In the **Redirect URIs** section, enter the following redirect URIs.
+     * `http://localhost:5000/redirect`
+   * Click on the **Add a platform** button in the **Platform configurations** section of the page.
+     * Select the **Single-page application** button and enter `https://localhost:5000` as the **Redirect URI** and click the **Configure** button.
+   * Under the **Implicit grant and hybrid flows** section enable both the **Access tokens** and **ID tokens** options.
```

Comment:
```suggestion
   * Under the **Implicit grant and hybrid flows** section enable both the **Access tokens** and **ID tokens** options.
```

Do we actually use implicit flow and/or hybrid flow in this sample? I think this bit needs to go.

Reply:
done!